certified-connectors/Team Forms/apiProperties.json

```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
```

Comment:
@AnthonyPhan please follow our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json) for your apiProperties.json. You can replace the secrets with dummy data.

Reply:
Thank you, fixed.